package.json

```diff
@@ -33,5 +20,27 @@
     "Fabien O'Carroll <fabien@allou.is>",
     "Kirill Fomichev <fanatid@ya.ru>"
   ],
-  "license": "MIT"
+  "files": [
+    "index.js"
```

Comment:
I work on an Electron app, WebTorrent Desktop, and have to deal with this issue. We just have a long list of folder/file names to ignore when packaging the app. Patterns like `test`, `tests`, `*.md`, etc. are ignored. It's technically possible for this to break a package, but it's been reliable so far :)

Full list here: https://github.com/feross/webtorrent-desktop/blob/master/bin/package.js#L85

Also, it's worth noting that the savings we achieved with this paled in comparison to the size of Electron, which packages all of Chromium.